ros2nodl/ros2nodl/command/nodl.py

```diff
@@ -1,21 +1,21 @@
-# Copyright 2020 Canonical Ltd.
```

Comment:
This is still public API. Does it need to be?

Reply:
fixed in 70d6ba5